docs/integrations/kubeflow.md

```diff
@@ -0,0 +1,97 @@
+* `LAKEFS_SECRET_ACCESS_KEY: <Secret access key>`
+* `LAKECTL_SERVER_ENDPOINT_URL: <Server endpoint URL>`
+
+### Example operations
+{: .no_toc }
+1. Create new branch 
+    
+   ```python
+    import kfp
+    from kfp import dsl
+    from kubernetes.client.models import V1EnvVar
+    
+    def create_branch_op():
+       return dsl.ContainerOp(
+          name='create_branch',
+          image='treeverse/lakectl',
+          arguments=['branch', 'create', 'lakefs://example-repo/example-branch', '-s', 'lakefs://example-repo/main']).add_env_variable(V1EnvVar(name='LAKECTL_CREDENTIALS_ACCESS_KEY_ID',value='<Access key ID>')).add_env_variable(V1EnvVar(name='LAKECTL_CREDENTIALS_SECRET_ACCESS_KEY',value='<Secret access key>')).add_env_variable(V1EnvVar(name='LAKECTL_SERVER_ENDPOINT_URL',value='<Server endpoint URL>'))
+    ```
+2. Commit changes to a branch
+    
+    ```python
+    def commit_op():
+        return dsl.ContainerOp(
+           name='commit',
+           image='treeverse/lakectl',
+           arguments=['commit', 'lakefs://repo/example-repo', '-m', 'commit message']).add_env_variable(V1EnvVar(name='LAKECTL_CREDENTIALS_ACCESS_KEY_ID',value='<Access key ID>')).add_env_variable(V1EnvVar(name='LAKECTL_CREDENTIALS_SECRET_ACCESS_KEY',value='<Secret access key>')).add_env_variable(V1EnvVar(name='LAKECTL_SERVER_ENDPOINT_URL',value='<Server endpoint URL>'))
+    ```
+3. Merge two lakeFS branches
+    
+   ```python
+    def merge_op():
+        return dsl.ContainerOp(
+           name='merge',
+           image='treeverse/lakectl',
+           arguments=['merge', 'lakefs://repo/example-branch', 'lakefs://example-repo/main']).add_env_variable(V1EnvVar(name='LAKECTL_CREDENTIALS_ACCESS_KEY_ID',value='<Access key ID>')).add_env_variable(V1EnvVar(name='LAKECTL_CREDENTIALS_SECRET_ACCESS_KEY',value='<Secret access key>')).add_env_variable(V1EnvVar(name='LAKECTL_SERVER_ENDPOINT_URL',value='<Server endpoint URL>'))
+    ```
+
+You can invoke any lakeFS operation supported by `lakectl` by implementing it a ContainerOp. Check out the complete [CLI reference](https://docs.lakefs.io/reference/commands.html) for the list of supported operations.  
+
+## Add the lakeFS steps to your pipeline
+Add the steps created on the previous step to your pipeline before compiling it. 
+
+### Example pipeline
+{: .no_toc }
+A pipeline that implements a simple ETL, that has steps for branch creation and commits.    
+   ```python
+    def lakectl_pipeline():
+       create_branch_task = create_branch_op()
+       extract_task = extract_op() # a made-up generic operation 
+       commit_task = commit_op() 
+       transform_task = transform_op() # a made-up generic operation
+       commit_task = commit_op()
+       load_task = load_op() # a made-up generic operation
+   ```
+
+## What's next?
+{: .no_toc }
+We are working to make the lakeFS [python client](https://docs.lakefs.io/integrations/python.html) work with Kubeflow.
+
+
+
+**Notes**  \  
```

Comment:
```suggestion
**Note**
```

Reply:
removed plural but kept the backslash to start a new line after "Note"